src/configs/base.ts

```diff
@@ -164,16 +164,34 @@ export const config: Config = {
   onSurface: [
     {
       name: 'border',
-      description: 'For use as a border on non-interactive elements.',
+      description: 'For use as the default border on elements.',
       light: {lightness: 60},
       dark: {lightness: 35},
       meta: {
         figmaName: 'Border/Default',
       },
     },
+    {
+      name: 'borderHovered',
```

Comment:
No need for a `borderPressed`?

Reply:
Hm, good question. I will check on that.

Reply:
Doesn't look like we need `borderPressed` right now